includes/class-wc-payments-features.php

```diff
@@ -350,6 +351,15 @@ public static function is_account_details_enabled(): bool {
 		return '1' === get_option( self::ACCOUNT_DETAILS_FLAG_NAME, '0' );
 	}
 
+	/**
+	 * Checks whether the Agentic Commerce feature is enabled.
+	 *
+	 * @return bool
+	 */
+	public static function is_agentic_commerce_enabled(): bool {
```

Comment:
If we have a feature flag in WooCommerce core, do we also need a feature flag in WooPayments?